src/lib/holocene/pagination.svelte

```diff
@@ -37,6 +38,14 @@
     }
   });
 
+  const handlePageChange = () => {
```

Comment:
Wondering if we want to also call `handlePageChange` `onMount` or not? It doesn't seem necessary if the first page is always going to be `1`.  However, if a `startingIndex` has been defined for the `Pagination` component the component loads to the corresponding page (which can be something other than `1`) and there is no `page` defined in the search params.

Reply:
What's the difference between the startingIndex and the page? Just trying to understand what we would using the startingIndex for

Reply:
Oh I see it's basically the same thing, it's just that you set the starting page with a prop. Yeah I think what you could do is an onMount and if startingIndex is not 0, update the currentPage and call handlePageChange.

Reply:
Updated ✅

Reply:
👍